pkg/scaffold/v1/resource/addtoscheme.go

```diff
@@ -48,16 +48,19 @@ func (a *AddToScheme) Validate() error {
 	return a.Resource.Validate()
 }
 
+// NB(directxman12): we need that package alias on the API import otherwise imports.Process
+// gets wicked (or hella, if you're feeling west-coasty) confused.
```

Comment:
Aah, I see it :), ignore my comment in one of the reviews about adding this note.